documentation/create_overview_doc/README.md

```diff
@@ -19,17 +19,17 @@ This powerful tool streamlines the creation of high-level documentation for soft
 
 - **Auto-Retry Logic and Enhanced Error Handling**: Ensures reliable documentation generation even in unstable environments through sophisticated error handling and retry mechanisms.
 
-- **Visualization with Mermaid.js Flow Maps and Enhanced Syntax Handling**: AI-generated visual flow maps to represent module interactions. Now with improved handling of Mermaid diagram syntax, minimizing manual interventions.
-
-- **Mermaid CLI Image Conversion**: Transforms Mermaid.js diagrams into images, offering a clear visual representation of your code architecture.
+- **Visualization with Code2Flow**: AI-generated visual flow maps to represent module interactions, configured with flags such as --hide-legend, --no-trimming, --verbose, and --skip-parse-errors for optimal detail and clarity.
```

Comment:
**Suggestion**: Ensure that the new flags introduced with Code2Flow ('--hide-legend', '--no-trimming', '--verbose', and '--skip-parse-errors') are well-documented in the README file. It's important for users to understand what each flag does and when to use them to generate flowcharts that best fit their needs. <br> **Code Suggestion**: 
 ```
 - **Visualization with Code2Flow**: AI-generated visual flow maps to represent module interactions. Configured with flags such as:
   - '--hide-legend': Hides the legend for a cleaner look.
   - '--no-trimming': Prevents trimming of node labels for full visibility.
   - '--verbose': Provides detailed logs during flowchart generation.
   - '--skip-parse-errors': Skips files with parsing errors, allowing the process to continue with other files.
 These flags are designed to optimize the detail and clarity of flowcharts.
 ```